lib/rpc.ts

```diff
@@ -160,7 +160,7 @@ export class Rpc extends EventEmitter {
       // TODO Test, then explain how this works.
       return new Proxy({}, {
         get: (target, property: string, receiver) => {
-          return (...args: any[]) => this._makeCall.bind(this, name, property, args, anyChecker);
+          return (...args: any[]) => this._makeCall.bind(this, name, property, args, anyChecker)();
```

Comment:
Oh, my version was certainly wrong, but this is overkill: it needs either `(...args) => this._makeCall(...)` or `this._makeCall.bind(...)`. I think.